distributed/bokeh/plots/status.py

```diff
@@ -0,0 +1,102 @@
+from bisect import bisect
+
+from bokeh.models import (
+    ColumnDataSource, Plot, Datetime, DataRange1d, Rect, LinearAxis,
+    DatetimeAxis, Grid, BasicTicker, HoverTool, BoxZoomTool, ResetTool,
+    PanTool, WheelZoomTool, Title
+)
+
+from distributed.utils import log_errors
+
+class Element(object):
+    """ Base class for Distributed UI Dashboard elements. """
+
+    def __init__(self):
+        """
+        Should be overridden by subclasses.
+
+        self.source ColumnDataSource
+        self.root should a layout-able object, a Plot, Layout, Widget or WidgetBox
+        """
+        self.source = None
+        self.root = None
```

Comment:
The term `root` wasn't immediately obvious to me.  Is this the standard Bokeh term for a layout-able object?  Something like `plot` might be more suggestive if it's not also incorrect in this case.